third_party/nvidia/hopper/lib/Transforms/WarpSpecialization/Utility.cpp

```diff
@@ -18,9 +18,11 @@ SmallVector<AsyncTaskId> getAsyncTaskIds(Operation *op) {
           asyncTaskIds[asyncTaskIds.size() - 1] != asyncTaskId)
         asyncTaskIds.push_back(asyncTaskId);
     }
+  } else if (auto attr = op->getAttrOfType<IntegerAttr>("ttg.partition")) {
+    int64_t idx = attr.getInt();
+    if (idx >= 0)
```

Comment:
For my understanding what does `< 0` represent? I'm assuming this is because of some special `-1` use case.

Reply:
Actually we should never have a `<0` case, but given the attribute is for a signed integer, I'm making that check.